styleguide/components/31-molecules/teaser/README.md

```diff
@@ -1,17 +1,37 @@
 # Teaser
 
-## Functionality
+## When to use this component
 
-- The entire teaser block is clickable.
-- Links and buttons inside the teaser are clickable.
+Use the teaser component in a <a href="{{path './collection.html'}}">collection</a> to show teasers of content on overview pages.
+
+## When not to use this component
+
+Do not use the teaser component for other uses than in a <a href="{{path './collection.html'}}">collection</a> on overview pages.
```

Comment:
A teaser could be used at the bottom of some other content pages as well, such as related articles. That implies that you **can** use it on other pages besides an overview page.

Reply:
Correct, I updated the Teaser documentation to better reflect this.